test/e2e/data/infrastructure-docker/v1beta1/v1.3/bases/cluster-with-kcp.yaml

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
general comment on v1beta1\vx folders.
we need to keep them documented in the readme that exists int v1beta1 folder, and possibly be a little bit more specific about why we need those version-specific templates and when we can drop them.
note: adding those additional details could be a follow up PR, but at least let's document them

Reply:
Issue for tracking this: https://github.com/kubernetes-sigs/cluster-api/issues/7699

Reply:
I'll follow-up on the issue and on this PR. I tried to document how it works in the release team doc + in the README.md but it looks like we have to improve the doc

Reply:
I also have some ideas how we can refactor the folder structure to make it easier to understand / maintain.